<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">Widget{
border-image: url(:/new/prefix1/picture/11.png);
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>211</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/picture/1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Pause_PushButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>310</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/picture/play.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Up_PushButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>310</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/picture/up.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Next_PushButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>310</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/picture/next.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Fast_PushButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>310</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/picture/fast.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Low_PushButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>310</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/picture/low.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Volume_PushButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>320</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/picture/volume.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>180</y>
     <width>131</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/picture/2.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Table_PushButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>330</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/picture/no_show.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="Song_TextEdit">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>210</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/picture/song.png);
background-repeat:no-repeat;              
background-position:center;              
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
  </widget>
  <widget class="QTextEdit" name="Get_Time_Length_TextEdit">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>260</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/picture/8.png);
background-repeat:no-repeat;              
background-position:center;     </string>
   </property>
  </widget>
  <widget class="QTextEdit" name="Get_Time_Pos_TextEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>260</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/picture/7.png);
background-repeat:no-repeat;              
background-position:center;  </string>
   </property>
  </widget>
  <widget class="QProgressBar" name="ProgressBar">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>370</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/picture/processBar1.png);
background-repeat:no-repeat;              
background-position:center;              </string>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="textDirection">
    <enum>QProgressBar::TopToBottom</enum>
   </property>
   <property name="format">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Mule_PushButton">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>110</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/picture/mule.png);
background-repeat:no-repeat;              
background-position:center;              </string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="List_Song_PushButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>160</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/picture/list_song.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>270</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/picture/18.png);            
background-position:center; </string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>10</y>
     <width>131</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/picture/17.png);
background-repeat:no-repeat;              
background-position:center;  </string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>-40</x>
     <y>410</y>
     <width>121</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/picture/28.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>-80</x>
     <y>-20</y>
     <width>201</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/picture/27.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>160</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/picture/23.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>370</y>
     <width>151</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/picture/26.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Like_Listwidget_PushButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>280</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/picture/like2.png);
background-repeat:no-repeat;              
background-position:center;    </string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Like_PushButton">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>330</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/picture/zan.png);
background-repeat:no-repeat;              
background-position:center;   </string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>Pause_PushButton</zorder>
  <zorder>Up_PushButton</zorder>
  <zorder>Next_PushButton</zorder>
  <zorder>Fast_PushButton</zorder>
  <zorder>Low_PushButton</zorder>
  <zorder>Volume_PushButton</zorder>
  <zorder>Song_TextEdit</zorder>
  <zorder>Get_Time_Length_TextEdit</zorder>
  <zorder>Get_Time_Pos_TextEdit</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>ProgressBar</zorder>
  <zorder>Mule_PushButton</zorder>
  <zorder>List_Song_PushButton</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>Table_PushButton</zorder>
  <zorder>label_3</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
  <zorder>Like_Listwidget_PushButton</zorder>
  <zorder>Like_PushButton</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
